Kubernates logo link

arunvelsriram opened the pull request:
Made Kubernates logo a link to dashboard home page. This helps dashboard users to go to home page directly from any other page.

floreks commented:
Adding `<a href="/"...` causes flickering and state reload. I'd use `ui-sref=overview` instead. Overview should be returned by controller method. Also it needs css class to change cursor icon when hovering over logo but usability is better.

arunvelsriram commented:
@floreks Agreed. I have changed to `ui-sref="overview"` and added a class.

Am not quite sure what do you mean by "Overview should be returned by controller method".  Can you please throw some light on that?

arunvelsriram commented:
I have made state name "overview" to be returned from a controller method. Hope my understanding is correct.

Will squash the commits and push again once you confirm.

One question: My changes have introduced a fall in the test coverage. I think there nothing impactful to test. Do you have any suggestion on that?

floreks commented on src/app/frontend/chrome/chrome.html:
```diff
@@ -17,12 +17,14 @@
-      <md-icon md-svg-icon="assets/images/kubernetes-logo-text.svg"
-               class="kd-toolbar-logo-text">
-      </md-icon>
+      <a ui-sref="{{$ctrl.overviewState()}}" class="kd-toolbar-logo-link">
```
1. `ui-sref` does not have to be placed on `a` tag. Move it to the `div` above and remove `a`.
2. Class `kd-toolbar-logo-link` does not exist. You have to add it to `chrome.scss`.

arunvelsriram replied:
1. `ui-sref` binds `a` tag to as far as I know. I used [this](https://github.com/angular-ui/ui-router/wiki/Quick-Reference#ui-sref) for reference
2. Will do it once you confirm the above

floreks replied:
I have checked and it also works on other tags. Only difference is that you need to style it manually.

arunvelsriram replied:
Yeah. But it makes the whole `kd-logo-bar` clickable. I mean the empty space between the logo text and search bar. We could apply some style to prevent that. But do you want the whole `kd-logo-bar` to be clickable?

floreks replied:
I think it's ok. It is considered as a single element.

floreks replied:
Ahh, this element is very wide. Then it's ok to add extra tag to wrap icons.

floreks commented on src/app/frontend/chrome/component.js:
```diff
@@ -102,6 +103,13 @@ export class ChromeController {
+  /**
+   * @export
+   */
+  overviewState() {
```
s/overviewState/getOverviewStateName

floreks commented on src/app/frontend/chrome/component.js:
```diff
@@ -102,6 +103,13 @@ export class ChromeController {
+
+  /**
+   * @export
+   */
```
Add `@return {string}`
